vue3-apollo-quasar/src/components/RepoAbout/HomepageUrl.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div v-if="homepageUrl" class="linkContainer">
+    <svg
```

Comment:
Please check the Readme about how to use custom icon with quasar its pretty easy. This is to avoid us using the svg this way and make the icon usable in case we need to.